src/main/java/mate/project/service/BookService.java

```diff
@@ -4,11 +4,12 @@
 import mate.project.dto.BookDto;
 import mate.project.dto.BookSearchParameters;
 import mate.project.dto.CreateBookRequestDto;
+import org.springframework.data.domain.Pageable;
 
 public interface BookService {
     BookDto save(CreateBookRequestDto createBookRequestDto);
 
-    List<BookDto> findAll();
+    List<BookDto> findAll(Pageable pageable);
```

Comment:
The 'findAll' method has been changed to accept a 'Pageable' parameter, but there is no handling of the case when this parameter is null. It would be better to provide two separate methods, one without parameters and one with the 'Pageable' parameter, to avoid null handling.